week-03/main.js

```diff
@@ -12,25 +13,37 @@ var list = [1, 2, 3, 4, 5];
 // `isAlive` - a boolean,
 // and `hobbies` an array.
 // Assign the object to the variable `user`
-
+user = {
+  name: 'Sam',
+  isAlive: true,
+  hobbies: ['running', 'eating'];
+}
 
 
 // Add 'golf' to the `user`'s `hobby` array.
-
+user.hobbies.push('golf');
 
 // # Functions
 
 // Define a function `sayHello` that takes a name string and returns "Hello" plus that name and an exclamation point.
 // Example:
 
 // sayHello('Mer') // "Hello Mer!"
+var sayHello = function(name) {
+  var name = 'Sally';
```

Comment:
This will only return `Sally` for all function calls. You're disregarding the parameter name.